bedrock/firefox/views.py

```diff
@@ -308,28 +227,6 @@ def firefox_os_geo_redirect(request):
     return HttpResponseRedirect(reverse('firefox.os.ver.{0}'.format(version)))
 
 
-@csrf_protect
-def firefox_partners(request):
-    # If the current locale isn't in our list, return the en-US value
-    press_locale = request.locale if (
-        request.locale in LOCALE_FXOS_HEADLINES) else 'en-US'
-
-    template_vars = {
-        'locale_headline_url': LOCALE_FXOS_HEADLINES[press_locale]['url'],
-        'locale_headline_title': LOCALE_FXOS_HEADLINES[press_locale]['title'],
-        'js_common': JS_COMMON,
-        'js_mobile': JS_MOBILE,
-        'js_desktop': JS_DESKTOP,
-    }
-
-    form_kwargs = {
-        'interest_set': 'fx',
-        'lead_source': 'www.mozilla.org/firefox/partners/'}
```

Comment:
Agree on retaining the use of this feature, will defer to yourself and @pmclanahan on what should be done (if anything) :+1: